### ISSUE TITLE
Azure App Service

### ISSUE THREAD

**Issue by wahmedswl** [hashicorp/waypoint#1361]
**Is your feature request related to a problem? Please describe.**
Nope, this is request to add support Azure App Service platform.

**Describe the solution you'd like**
Waypoint should be able to deploy to App Service, might be with reduced feature set.

**Describe alternatives you've considered**
Pulumi, currently used that for Automation and deployment as well. It lacks the universal nature of Waypoint and single tool for heterogenous environments for developers.

**Explain any additional use-cases**

**Additional context**

Thanks

**Comment by magodo** [comment]
@wahmedswl I just created a [waypoint plugin for app service](https://github.com/magodo/waypoint-plugin-azure-app-service), maybe you wanna take a look?

**Comment by wahmedswl** [comment]
Hi, @magodo awesome 👍 from the quick look, its seems that its supporting container right now, is it? App Service support direct deploy from Storage Account etc, is that will also be supported.

Thanks

**Comment by magodo** [comment]
@wahmedswl Yes, currently it only supports container. Would you mind open an issue in the plugin repo for the feature you wanted?
